src/02-common-principles.md

```diff
@@ -125,6 +125,25 @@ The solutions will change from case to case and publicly available datasets will
 be reviewed to include common data types in the future releases of the BIDS
 specification.
 
+### Metadata conflicts
+
+BIDS specifies that metadata be stored in human-readable formats such as
+TSV (see [Tabular files](#tabular-files)) or
+JSON (see [Key/value files](#keyvalue-files-dictionaries)).
+In some cases, this duplicates metadata internal to the imaging file.
```

Comment:
applicable not only to "imaging" data AFAIK thus (similar to #760 somewhat) might need a more generic wording here.  Meanwhile:
```suggestion
In some cases, this duplicates metadata internal to the data sample file(s).
```
and made it `(s)` since there could be those `_part-` and smth else which would correspond to the same "data sample"...

Reply:
I like "data file" more than "data sample file" (as you proposed in https://github.com/bids-standard/bids-specification/pull/761#discussion_r596093857). The pairing of "data file" and "metadata file" is clearer IMHO.